src/js/components/presentational/ButtonStyle.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const StyledButton = styled.button`
+width: ${({width}) => (width ? `${width}px` : '50px')};
+height: ${({ height}) => (height ? `${height}px` : '25px')};
+color: ${({color}) => (color ? color:'blue')};
+cursor: ${({cursor}) => (cursor ? cursor: 'pointer')};
+name: ${({name}) => (name ? name:'Buttone')};
+background: ${({background}) => (background ? background:'lightblue')};
+label: ${({label}) => (label ? label:'inherit')};
```

Comment:
remove label, name, value, background => background-color, add a new line to the end, and as with Matt, lets switch from using pixels to 1-2 (or maybe even 3) rem for width/height.

Reply:
Done, with the exception of 'background', which is actually what it was supposed to be already.